### ISSUE TITLE
Node-gyp rebuild fails

### ISSUE THREAD

**Issue by JoeBanks13** [octalmage/robotjs#266]
## Expected Behavior
I'm trying to install RobotJS onto my Windows 10 Laptop and it's just giving a syntax error

## Current Behavior
So I've been trying to install robotjs and I keep getting this error:
```
C:\Users\joebe_000\Dinosaur>npm install --save robotjs --python="C:/Windows/py.exe"

> robotjs@0.3.7 install C:\Users\joebe_000\Dinosaur\node_modules\robotjs
> node-gyp rebuild


C:\Users\joebe_000\Dinosaur\node_modules\robotjs>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "" rebuild )
Traceback (most recent call last):
  File "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\gyp\gyp_main.py", line 13, in <module>
    import gyp
  File "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\gyp\pylib\gyp\__init__.py", line 37
    print '%s:%s:%d:%s %s' % (mode.upper(), os.path.basename(ctx[0]), ctx[1], ctx[2], message)
                         ^
SyntaxError: invalid syntax
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\configure.js:305:16)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
gyp ERR! System Windows_NT 10.0.10586
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\joebe_000\Dinosaur\node_modules\robotjs
gyp ERR! node -v v6.8.1
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok
npm WARN enoent ENOENT: no such file or directory, open 'C:\Users\joebe_000\Dinosaur\node_modules\build\package.json'
npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "--save" "robotjs" "--python=C:/Windows/py.exe"
npm ERR! node v6.8.1
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE

npm ERR! robotjs@0.3.7 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the robotjs@0.3.7 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the robotjs package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs robotjs
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls robotjs
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\joebe_000\Dinosaur\npm-debug.log
```

I only installed Python 2.7.3 this morning after it was required for node-gyp, but I have python3.5 using the `python` alias on my PC, to get python 2.7.3 I have to use `py.exe file.py`

## Steps to Reproduce (for bugs)

1. npm install --save --python="C:/Windows/py.exe" robotjs 

## Context
Just not installing

## Your Environment

* RobotJS version: Current
* Node.js version: v6.8.1
* npm version: v3.10.8
* Operating System: Windows 10

**Comment by octalmage** [comment]
This should be resolved by v0.4.6, please let me know if you still encounter this and we can reopen the issue.

**Comment by codegeek2014** [comment]
I am trying to run `py gyp_main.py` still the same syntax error message. I modified the `gyp` and `gyp.bat` file and changed `exec python "${base}/gyp_main.py" "$@"` to `exec py "${base}/gyp_main.py" "$@"` to make it use python 2. But the issue still persists.
